kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/aggregation/settings/KoverParametersProcessor.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.aggregation.settings
+
+import kotlinx.kover.gradle.aggregation.settings.dsl.intern.KoverSettingsExtensionImpl
+import org.gradle.api.provider.HasMultipleValues
+import org.gradle.api.provider.ProviderFactory
+
+
+internal object KoverParametersProcessor {
+    fun process(settingsExtension: KoverSettingsExtensionImpl, providers: ProviderFactory) {
+        if (providers.gradleProperty("kover").isPresent) {
```

Comment:
Does it make sense to check support values for this property? For instance, suppose someone has a project where coverage was enabled using `enableCoverage` DSL's call, but a user now want's to explicitly disable it via CLI by passing `-Pkover=false`.

Reply:
> the users can write their own condition

That requires committing some code to a repo, so if someone want to temporary disable code coverage collection in CI, it could be problematic.

Reply:
in this case, we can add only `-Pkover=false`, so that there is `-Pkover` to turn on and `-Pkover=false` to turn off